tools/c7n_mailer/c7n_mailer/sqs_queue_processor.py

```diff
@@ -168,8 +164,17 @@ def process_sqs_messsage(self, encoded_sqs_message):
         sns_message_packages = sns_delivery.get_sns_message_packages(sqs_message)
         sns_delivery.deliver_sns_messages(sns_message_packages, sqs_message)
 
+        # this section sends a notification to the resource owner via Slack
+        if self.config.get('slack_token'):
+            from .slack_delivery import SlackDelivery
+            slack_delivery = SlackDelivery(self.config, self.session, self.logger)
+            if slack_delivery.is_deliverable(sqs_message):
+                slack_messages = slack_delivery.get_to_addrs_slack_messages_map(sqs_message, email_delivery)
+                slack_delivery.slack_handler(sqs_message, slack_messages)
+
         # this section gets the map of metrics to send to datadog and delivers it
-        if HAVE_DATADOG:
+        if self.config.get('datadog_api_key'):
```

Comment:
Probably it's not the best way to know if there is a Datadog Message, becaure if the resource that runs the mailer has the Datadog Agent installed don't need the datadog api key, probably a better option is see if there is any `to` with `datadog`.